content/en/references/configuration.md

```diff
@@ -147,48 +147,54 @@ The OpenSearch configuration variables are used to manage both, OpenSearch and E
 | `KINESIS_LATENCY` | `500` (default), `0` (to disable)| Integer value of milliseconds, causing the Kinesis API to delay returning a response in order to mimic latency from a live AWS call. |
 | `KINESIS_INITIALIZE_STREAMS` | `"my-first-stream:1,my-other-stream:2:us-west-2,my-last-stream:1"` | A comma-delimited string of stream names, its corresponding shard count and an optional region to initialize during startup. If the region is not provided, the default region is used. Only works with the `kinesis-mock` `KINESIS_PROVIDER`. |
 
-### Lambda (New / `v2`)
+### Lambda
+
+The [new lambda provider]({{< ref "references/lambda-v2-provider" >}}) `v2` (formerly known as `asf`) is active by default starting with Localstack&nbsp;v2.0.
 
 | Variable| Example Values | Description |
 | - | - | - |
```

Comment:
@dominikschubert @dfangl Should we add any of these to the public API?
* `LAMBDA_DOWNLOAD_AWS_LAYERS` (pro)
* `LAMBDA_RETRY_BASE_DELAY_SECONDS`
* `LAMBDA_LIMITS_*` (many)

Reply:
Added `LAMBDA_DOWNLOAD_AWS_LAYERS` (pro) and `LAMBDA_LIMITS_CONCURRENT_EXECUTIONS` as discussed with @dominikschubert
Others are kept internal / communicated selectively in Github issues on demand for now until we see further demand.